XCUITests/FxScreenGraph.swift

```diff
@@ -863,7 +863,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.press(reloadButton, to: ReloadLongPressMenu)
         screenState.tap(reloadButton, forAction: Action.ReloadURL, transitionTo: WebPageLoading) { _ in }
         } else {
-            let reloadButton = app.buttons[AccessibilityIdentifiers.Toolbar.reloadButton]
+            let reloadButton = app.buttons["Reload"]
```

Comment:
Curious as to how come we're adding strings, rather than using the identifiers? I noticed the identifier was a different string, sure, but wouldn't it be more future proof to update the ID of the button itself to use the correct a11yID?

Reply:
For some reason on iPad the reload button does not have an id. This is what you see about that button:
`Button, 0x7f87d3419900, {{98.0, 73.0}, {44.0, 44.0}}, label: 'Reload'` 
That's why I'm using there the reload, only for iPad, for iPhone I have not changed it.